runtime/src/main/resources/reference.conf

```diff
@@ -30,6 +30,9 @@ akka.grpc.client."*" {
   # Location on the classpath of CA PEM to trust
   trusted = ""
   use-tls = true
+  # SSL provider to use:
+  # leave empty to auto-detect, or configure 'jdk' or 'openssl'.
+  ssl-provider = ""
```

Comment:
Add `openssl_refcnt` on the comment? Not sure, TBH, the internal implementation is identical to `openssl`.